pkg/rclgo/example_action_server_type_safe_wrapper_test.go

```diff
@@ -0,0 +1,55 @@
+package rclgo_test
+
+import (
+	"context"
+	"errors"
+
+	example_interfaces_action "github.com/tiiuae/rclgo/internal/msgs/example_interfaces/action"
+	"github.com/tiiuae/rclgo/pkg/rclgo"
+)
+
+var typeSafeFibonacci = example_interfaces_action.NewFibonacciAction(
+	func(
+		ctx context.Context, goal *example_interfaces_action.FibonacciGoalHandle,
+	) (*example_interfaces_action.Fibonacci_Result, error) {
+		if goal.Description.Order < 0 {
+			return nil, errors.New("order must be non-negative")
+		}
+		sender, err := goal.Accept()
+		if err != nil {
+			return nil, err
+		}
+		logger := goal.Server().Node().Logger()
```

Comment:
Should there be a helper function to get the logger directly from the goal? `goal.GetLogger()` or just goal.Logger()

Reply:
Helper method added